portal-frontend/src/app/features/public/search/public-search.component.ts

```diff
@@ -193,6 +194,14 @@ export class PublicSearchComponent implements OnInit, OnDestroy {
     sessionStorage.setItem(SEARCH_SESSION_STORAGE_KEY, searchDto);
 
     this.isLoading = true;
+    const element = this.elementRef.nativeElement.querySelector(`#${CSS.escape('searchResults')}`);
```

Comment:
Remove the element ref, and use the ID you gave it with scrollToElement util

Reply:
Yeah, this is probably better since it keeps behaviour consistent.

Reply:
made changes